src/network_manager.rs

```diff
@@ -178,6 +182,8 @@ pub struct PeerNetManager<
     init_connection_handler: I,
     context: Ctx,
     transports: HashMap<TransportType, InternalTransportType<Id>>,
+    total_bytes_received: Arc<StdRwLock<u64>>,
```

Comment:
Why not use parking lot as we are using them in the rest of the code ?

Reply:
done